test/support/input.ex

```diff
@@ -0,0 +1,6 @@
+defmodule InputSupport do
+  def prepare(urls) do
```

Comment:
Maybe now when this is in a separate module, “prepare” is to vague. ”prepare_url_list“, “url_list”, “prepare_downloader_input”, “downloader_input”…

Reply:
Absolutely, thanks